webopenface/views.py

```diff
@@ -1,17 +1,35 @@
 from django.shortcuts import render
 from django.http import HttpResponse
-from webopenface.models import DetectedPeople, DetectedFace
+from webopenface.models import Person, DetectedPeople, DetectedFace, Frame
 
 
 def index(request):
-    preview_list = DetectedPeople.objects.order_by('face__frame__add_date')
-    output = ', '.join(i.person.name for i in preview_list)
-    return HttpResponse(output)
+    added_people = Person.objects.order_by('name')
+    context = {'added_people': added_people}
+    return render(request, 'webopenface/index.html', context)
```

Comment:
I would like to follow this convention: 
return render(
    request,
    'template.html',
    {
        'key': 'value',
    },
)